spec/controllers/hq/hq_controller_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+class NyuksController < HqController; end
+
+RSpec.describe HqController, type: :controller do
+  describe 'owns namespace controllers' do
+    Rails.application.eager_load!
+    ApplicationController.descendants.each do |_controller|
+      if _controller.to_s.include? 'Hq::'
+        specify "HqController is the parent of #{_controller.to_s}" do
+          expect(_controller.superclass).to eq HqController
+        end
+      end
+    end
+  end
+
+  controller NyuksController do
+    def foobar_action
+      render nothing: true
+    end
+  end
+
+  describe 'anonymous child' do
```

Comment:
Some sort of test description that explains we are checking for authentication or similar would be really helpful. I don't think someone new to the code base would find it easy to work out what this test is testing.